cdap-ui/app/features/admin/templates/namespace/create.html

```diff
@@ -15,7 +15,7 @@
 -->
 
 <div class="modal-header clearfix">
-  <h3 class="modal-title pull-left"> Add Namespace </h3>
+  <h3 class="modal-title pull-left"> Add a Namespace </h3>
```

Comment:
I think "Add Namespace" is better since we usually have "Add X" instead of "Add a/an X"